console/Makefile

```diff
@@ -1,9 +1,14 @@
 include ../Makefile.common
 
-build:
-	npm install
+build: npm_install www/tooltips.json www/help.html
 
-package:
+www/tooltips.json: ../doc-utils/parse.py ../doc-utils/expected.json ../documentation/modules/*
+	../doc-utils/parse.py -i ../doc-utils/expected.json -o www/tooltips.json -v ../documentation/modules/
+
+www/help.html:  ../documentation/master.adoc ../documentation/modules/*.adoc ../documentation/common/*.adoc
+	asciidoctor -d book -B ../documentation/ -o ../console/www/help.html ../documentation/master.adoc
+
```

Comment:
presumably we should add the requirement for asciidoctor to HACKING.md?

Reply:
And mark it as 'optional' in the sense that you only need it to build the docs, which I think some devs won't be interested in building.